### PR TITLE
Update for RDS

### PR DESCRIPTION
**Pull request by scottslowe**
This PR updates the Pulumi program to add support for RDS (exports the ID of the security group created for the nodes).

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+const baseOrgName = config.get("baseOrgName") || "zephyr";
+const baseProjName = config.get("baseProjName") || "zephyr-infra";
+const baseStackName = config.get("baseStackName") || currentStackName;
 
 // Create a StackReference to get Kubeconfig from base stack
-const infraSr = new pulumi.StackReference(`${infraOrgName}/${infraProjName}/${infraStackName}`);
-const infraVpcId = infraSr.getOutput("vpcId");
-const infraPrivSubnetIds = infraSr.getOutput("privSubnetIds");
-const infraPubSubnetIds = infraSr.getOutput("pubSubnetIds");
+const baseSr = new pulumi.StackReference(`${baseOrgName}/${baseProjName}/${baseStackName}`);
```
Comment (ringods) [inline index.ts:18]: Assuming we deploy the different layered stacks in the same organization, and keep the stack name consistent for stacks making up the same environment (e.g. `dev`), wouldn't it be better to showcase the `getOrganization` and `getStack` calls from our core SDK?

https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/pulumi/#getOrganization
https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/pulumi/#getStack

File: index.ts [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
     maxSize: maxClusterSize,
     nodeAssociatePublicIpAddress: false,
-    // Uncomment the next two lines for a private cluster (VPN access required)
-    // endpointPrivateAccess: true,
-    // endpointPublicAccess: false
 });
 
 // Export some values for use elsewhere
 export const kubeconfig = eksCluster.kubeconfig;
+export const nodeSecurityGrp = eksCluster.nodeSecurityGroup.id;
```
Comment (ringods) [inline index.ts:37]: Nitpicking, but I would prefer to have `Group` instead of `Grp` in the output name.

**Review by ringods — CHANGES_REQUESTED** [review]

File: index.ts (hunk 1 above)
Comment (ringods) [reply index.ts:18]: Overlooked you already use `getStack`. Remark still stands for `getOrganization` though.

Comment (scottslowe) [reply index.ts:21]: Resolved in bef672c. I added `getOrganization` to provide a default organization name if the user does not supply an organization name in the stack configuration. Great idea!

File: index.ts (hunk 2 above)
Comment (scottslowe) [reply index.ts]: Changed in bef672c per your suggestion

**Pull request merged**